### ISSUE TITLE
.apply behaviour changing

### ISSUE THREAD

**Issue by jklenzing** [rstoneback/pysat#160]
```
/home/travis/build/rstoneback/pysat/pysat/instruments/omni_hro.py:222: FutureWarning: 
Currently, 'apply' passes the values as ndarrays to the applied function. In the future, this 
will change to passing it as Series objects. You need to specify 'raw=True' to keep the 
current behaviour, and you can pass 'raw=False' to silence this warning
  center=True).apply(pysat.utils.nan_circstd, kwargs=circ_kwargs)
```

**Comment by jklenzing** [comment]
From https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.window.Rolling.apply.html

Possible values for `raw`:

  - False : passes each row or column as a Series to the function.
  - True or None : the passed function will receive ndarray objects instead. If you are just applying a NumPy reduction function this will achieve much better performance.

raw=True will maintain the current behavior.  I propose this option unless we have good reason to change.  Pinging @rstoneback and @aburrell.

**Comment by jklenzing** [comment]
4 results (1 commented) found in develop branch.

<img width="719" alt="screen shot 2019-02-21 at 9 40 40 pm" src="https://user-images.githubusercontent.com/19592220/53216079-60f4fe00-3621-11e9-92ef-3c803b3321c8.png">

**Comment by jklenzing** [comment]
Closed with merge of #173.
